src/lib/multiply/index.ts

```diff
@@ -0,0 +1,91 @@
+import {PluginInterface} from '../../interfaces';
+import {PluginParams} from '../../types/common';
+import {buildErrorMessage} from '../../util/helpers';
+import {ERRORS} from '../../util/errors';
+import {MultiplyConfig} from './types';
+const {InputValidationError} = ERRORS;
+
+export const Multiply = (globalConfig: MultiplyConfig): PluginInterface => {
+  const errorBuilder = buildErrorMessage(Multiply.name);
+  const inputParameters = globalConfig.inputParameters || [];
```

Comment:
the `globalConfig` has `input-parameters` and `output-parameter` properties, please change to get correct properties and change in the MultiplyConfig type also